app/controllers/account/transactions_controller.rb

```diff
@@ -12,8 +12,7 @@ def index
   end
 
   def update
-    @entry.update!(entry_params.merge(amount: amount))
```

Comment:
Merging in `amount` caused the scenario when we didn't include the param, it was calling `nil.to_d`, which resulted in 0